spring-cloud-kubernetes-client-config/src/main/java/org/springframework/cloud/kubernetes/client/config/KubernetesClientConfigMapPropertySource.java

```diff
@@ -52,8 +52,7 @@ private static SourceData getSourceData(KubernetesClientConfigContext context) {
 	// we need to pass various functions because the code we are interested in
 	// is protected in ConfigMapPropertySource, and must stay that way.
 	private static KubernetesClientContextToSourceData namedConfigMap() {
-		return NamedConfigMapContextToSourceDataProvider.of(ConfigMapPropertySource::processAllEntries,
```

Comment:
these `ConfigMapPropertySource::getSourceName, ConfigMapPropertySource::withPrefix` are not needed anymore, and as matter of fact they do not exist either, moved to `ConfigUtils`, so that they could be reused in secrets and config maps